mail/dovecot2/Portfile

```diff
@@ -105,6 +105,82 @@ variant libstemmer description {Use libstemmer for full-text search} {
     configure.args-append       --with-libstemmer
 }
 
+variant apns \
+    description "Enable Apple Push Notification Service (APNS)" {
+    # diff -Naur core-2.3.7.1 core-2.3.7.1-pushnotify | sed -E -e 's/core-([[:digit:]]+\.?)+(-pushnotify)?\//.\//g' > dovecot-core-pushnotify.patch
+    patchfiles-append \
+        dovecot-core-pushnotify.patch
+    post-patch {
+        reinplace "s|@PREFIX@|${prefix}|g" \
+            ${worksrcpath}/src/imap/cmd-x-apple-push-service.c \
+            ${worksrcpath}/src/plugins/push-notify/push-notify-plugin.c
+    }
+    depends_run-append \
+                    port:p5-net-apns-persistent \
+                    port:p5-privileges-drop
```

Comment:
It is not correct to depend on a stub port. You must choose a specific version of perl, normally p5.28 currently.

Reply:
Thanks. Done.